packages/kubernetes-1.23/kubernetes-1.23.spec

```diff
@@ -229,4 +261,12 @@ install -p -m 0644 %{S:102} %{buildroot}%{_cross_templatedir}/pod-infra-containe
 %files -n %{_cross_os}kubelet-1.23-fips-bin
 %{_cross_fips_bindir}/kubelet
 
+%files -n %{_cross_os}kube-proxy-1.23
```

Comment:
Is this intentional?
If I am not mistaken, when there’s no list of files or directories after the %files directive, it typically means all files and directories that have been installed to the buildroot during the %install phase are automatically included

Reply:
Without the `%files` directive, the package itself wasn't created.  With this, its an empty package with only dependencies:

```
% rpm -qlp build/rpms/kubernetes-1.28/bottlerocket-kube-proxy-1.28-1.28.7-1.1719972209.77427686.br1.x86_64.rpm
(contains no files)

% rpm -qpR build/rpms/kubernetes-1.28/bottlerocket-kube-proxy-1.28-1.28.7-1.1719972209.77427686.br1.x86_64.rpm
bottlerocket-kube-proxy-1.28(binaries)
bottlerocket-kubelet-1.28
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsZstd) <= 5.4.18-1
```

Reply:
@monirul , this is intentional. In the FIPS world, the new way to compile go packages is by having the main package provide either systemd services, configurations or `tmpfiles.d` drop ins, and use the image feature flag `image-feature-fips`/`image-feature-no-fips` to control which binary is actually installed when the image is built.